packages/superset-ui-translation/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@superset-ui/translation",
+  "version": "0.0.0",
+  "description": "Superset UI translation",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "files": [
+    "esm",
+    "lib"
+  ],
+  "scripts": {
+    "build:cjs": "beemo babel ./src --out-dir lib/ --minify",
+    "build:esm": "beemo babel ./src --out-dir esm/ --esm --minify",
+    "build": "yarn run build:cjs && yarn run build:esm",
+    "dev": "beemo babel --watch ./src --out-dir esm/ --esm",
+    "jest": "beemo jest --color --coverage",
+    "eslint": "beemo eslint \"./{src,test}/**/*.{js,jsx,json,md}\"",
+    "lint": "yarn run prettier && yarn run eslint",
+    "lint:fix": "yarn run prettier --write && yarn run eslint --fix",
+    "test": "yarn run jest",
+    "prettier": "beemo prettier \"./{src,test}/**/*.{js,jsx,json,md}\"",
+    "prepublish": "yarn run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache-superset/superset-ui.git"
+  },
+  "keywords": [
+    "superset",
+    "client",
+    "translation"
+  ],
+  "author": "",
```

Comment:
I'm not sure what to do about these. whoever inits the package could put their name, or maybe we can omit the field in `package.json`s to make it a non-issue 🤔

Reply:
I'll put `Superset` for now.